### DISCUSSION TITLE
Front Ends iniciantes buscando trampo

### DISCUSSION THREAD

**Discussion by woliveiras** [frontendbr/forum#1030] [Dicas]
Galera, o @training-center possui um projeto para ajudar pessoas iniciantes a entrarem na área de desenvolvimento de software e temos algumas pessoas buscando trampo como Front End, quem puder ajudar, compartilhem ae nas empresas de vocês ou nas redes sociais: [Front Ends buscando o primeiro trampo](https://github.com/training-center/primeiro-trampo/issues?q=is%3Aissue+is%3Aopen+label%3A%22Front+End%22)

**Comment by ghost** [comment]
@woliveiras sempre ajudando a galera que tá começando. rsrs
